src/cpu/minor/execute.cc

```diff
@@ -876,6 +876,17 @@ Execute::doInstCommitAccounting(MinorDynInstPtr inst)
     cpu.commitStats[inst->id.threadId]->numOps++;
     cpu.commitStats[inst->id.threadId]
         ->committedInstType[inst->staticInst->opClass()]++;
+     if (inst->isInst())
+    {
+            if (inst->staticInst->isVector())
+            {
+                    cpu.executeStats[inst->id.threadId]->numVecAluAccesses++;
```

Comment:
It does not look like this code addition conforms to the spacing and formatting rules.  Did you run the commit hook for this?  Did you run the style.py script before pushing?

Reply:
I think I did, but I double-checked and it does look like it's not conforming to the style guide, will fix. Thanks for catching that!